src/app/features/home/post-capture-tab/post-capture-tab.component.html

```diff
@@ -23,13 +23,18 @@
         [routerLink]="['post-capture-details', { id: postCapture.id }]"
       >
         <ion-img [src]="postCapture.asset_file_thumbnail"></ion-img>
+        <mat-icon class="from-store">shopping_bag</mat-icon>
       </mat-grid-tile>
     </mat-grid-list>
   </div>
 
   <div class="series-wrapper" *ngSwitchCase="'Series'">
-    <app-series-card routerLink="series"></app-series-card>
-    <app-series-card routerLink="series"></app-series-card>
-    <app-series-card routerLink="series"></app-series-card>
+    <div
```

Comment:
Suggestion Only: Use `ion-card` to use built-in card style. Remember to remove unused CSS rules.